.circleci/config.yml

```diff
@@ -673,7 +673,7 @@ jobs:
       - run:
           no_output_timeout: 45m
           command: |
-            #export PATH=$(echo "$PATH" | sed -e 's|:/home/circleci/\.go_workspace/bin||g' | sed -e 's|:/usr/local/go/bin||g')
+            # export PATH=$(echo "$PATH" | sed -e 's|:/home/circleci/\.go_workspace/bin||g' | sed -e 's|:/usr/local/go/bin||g')
```

Comment:
why is this here?

Reply:
because ide added a space

Reply:
since we don't currently run that workflow (windows build) I can't say for sure, and don't want to remove it until we test if we need it or not.